README.rst

```diff
@@ -1,5 +1,13 @@
-p5
-===
+.. raw:: html
+
+   <h1 align="center">p5</h1>
+
+   <p align="center">
+      <img alt="PyPI" src="https://img.shields.io/pypi/l/p5?color=light-green">
+      <img alt="PyPI" src="https://img.shields.io/pypi/v/p5?color=blue">
+      <img alt="PyPI" src="https://img.shields.io/github/issues-raw/p5py/p5">
```

Comment:
Since Github already has an issues tab, maybe this isn't too useful?

Reply:
Yep 👍🏻